app/ide-desktop/lib/dashboard/src/authentication/src/dashboard/components/templates.tsx

```diff
@@ -1,62 +1,137 @@
+const genBackgroundImage = () => `url(${Math.random() > 0.5 ? dataUrl1 : dataUrl2})`
 
-  return (
-    <div className="bg-white">
-      <div className="mx-auto py-16 px-4 sm:py-24 sm:px-6 lg:px-8">
-        <div className="grid grid-cols-1 gap-y-10 gap-x-6 sm:grid-cols-3 lg:grid-cols-4 xl:grid-cols-5 xl:gap-x-8">
-          {templates.map((template, index) => (
-            <div
-              key={index}
-              className="h-40 rounded-lg border-4 border-dashed border-gray-200 flex"
-            >
-              <div className="m-auto text-center">
-                <button
-                  onClick={() => {
-                    onChange(template.templateName);
-                  }}
+/** Render single template */
+interface TemplateRenderProps extends TemplatesProps {
+    template: Template
+}
+// eslint-disable-next-line @typescript-eslint/naming-convention
+const TemplateRender: React.FC<TemplateRenderProps> = ({ template, onChange }) => {
+    /** Unify the border color and the text color */
+    const borderColor = '9E8C91'
```

Comment:
i would extract this to a tailwind `theme.extends` and then use the color name, but maybe it isn't easy on this branch

Reply:
This color is currently only used in this component. 
If used extensively, we can refine it into a theme later.